ngraph/python/tests/test_ngraph/test_dft.py

```diff
@@ -0,0 +1,320 @@
+        -3.8213987, -0.26443875, -1.6623533, -2.6665692, 2.6686997, -0.6977545, 5.85767,
+        -3.9102163, -11.673204, -2.3073153, -4.529278, 4.0891604, 3.9445055, 1.8883687,
+        1.50531, -7.2083244, 3.1367111, 1.1151649, -4.1500554, -0.54910004, -0.48040384,
+        11.444895, -2.6333811, -3.0142484, 4.6609726, 1.755743, 0.87769306, -0.7609439,
+        -0.26591438, 6.615961, -2.141545, -2.7914915, -4.2386503, 3.1565619, -6.6059103,
+        -7.35018, -2.2787585, 5.836963, -2.6666338, 0.98255026, 5.199881, 8.640279,
+        1.7439961, 2.191582, -4.535021, -5.038538, -0.841679, -6.8834453, -4.654301,
+        -0.220559, -4.7396717, -9.393296, 0.32385087, 3.9426038, -4.9187584, 1.7061774,
+        -4.8232145, -0.5627973, -2.3221302, -1.1155958, -2.7412212, 6.798079, -4.0860014,
+        1.9515686, 4.2942266, 0.5557329, -1.9789174, -4.973804, -2.0268555, -3.9974911,
+        -8.164038, 3.3319929, -2.474605, 0.39113098, 2.0651584, 5.5962815, -1.1102749,
+        -1.2390921, -5.0933027, -4.0492353, 5.009116, 3.323446, -1.0033474, -0.54384375,
+        -3.4698372, -2.3566747, -6.545992, 1.3816929, -2.0633929, -6.3665648, -4.13964,
+        -3.4099324, -1.1418146, 8.466255, 3.2365537, -0.14618888, 1.3563147, 0.3446387,
+        3.1233552, 0.7530624, 0.548483, -1.1876376, -8.070564, 1.4254899, -0.9140264,
+        2.5087235, -1.3091599, 0.9416502, 0.16097029, 2.6614356, 1.9558911, 4.219861,
+        1.1494511
+    ]
+    return np.array(expected_data, dtype=np.float32).reshape((2, 10, 10, 2))
+
+
+@xfail_issue_49359
+def test_dft_1d():
+    runtime = get_runtime()
+    input_data = build_fft_input_data()
+    input_tensor = ng.constant(input_data)
+    input_axes = ng.constant(np.array([2], dtype=np.int64))
+
+    dft_node = ng.dft(input_tensor, input_axes)
+    computation = runtime.computation(dft_node)
+    dft_results = computation()
+    expected_results = get_fft1d_expected_output()
+    assert np.allclose(dft_results, expected_results, atol=0.00001)
+
+
+@xfail_issue_49359
+def test_dft_2d():
+    runtime = get_runtime()
+    input_data = build_fft_input_data()
+    input_tensor = ng.constant(input_data)
+    input_axes = ng.constant(np.array([1, 2], dtype=np.int64))
+
+    dft_node = ng.dft(input_tensor, input_axes)
+    computation = runtime.computation(dft_node)
+    dft_results = computation()
+    expected_results = get_fft2d_expected_output()
+    assert np.allclose(dft_results, expected_results, atol=0.000062)
+
+
+@xfail_issue_49359
+def test_dft_3d():
+    runtime = get_runtime()
+    input_data = build_fft_input_data()
+    input_tensor = ng.constant(input_data)
+    input_axes = ng.constant(np.array([0, 1, 2], dtype=np.int64))
+
+    dft_node = ng.dft(input_tensor, input_axes)
+    computation = runtime.computation(dft_node)
+    dft_results = computation()
+    expected_results = get_fft3d_expected_output()
+    assert np.allclose(dft_results, expected_results, atol=0.0002)
```

Comment:
please add test with signal_size not None

Reply:
Done.